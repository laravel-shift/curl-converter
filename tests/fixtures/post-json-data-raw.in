curl -X POST --data-raw '{"foo":"bar"}' https://example.com
