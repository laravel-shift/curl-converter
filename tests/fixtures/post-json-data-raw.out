Http::withBody('{"foo":"bar"}')
    ->post('https://example.com');
